doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -5,6 +5,77 @@
 
 It's fairly easy to use, although it can be highly customized.
 
+== Installation Guide
+
+[#_where_to_get_the_jars]
+=== Where to get the JARs?
+
+You can download the connector JAR from the link:https://neo4j.com/product/connectors/apache-spark-connector/[Neo4j Connector Page] or from the link:https://github.com/neo4j-contrib/neo4j-spark-connector/releases[GitHub releases page].
+
+Choose the version carefully since the wrong combination of Scala version and Spark version will break your code.
+Check xref:overview.adoc#_spark_compatibility[this page] for more info about the proper JAR to use.
+
+=== Using spark-submit, spark-shell or pyspark
+
+[shell]
+----
+$SPARK_HOME/bin/spark-shell --jars neo4j-connector-apache-spark_2.12_3-4.0.1.jar
+----
+
+The connector is also available from link:https://spark-packages.org/?q=neo4j-connector-apache-spark[Spark Packages]:
+
+[shell]
+----
+$SPARK_HOME/bin/spark-shell --packages neo4j-contrib:neo4j-connector-apache-spark_2.12_3:4.0.1
+----
+
+=== Using sbt
+
+If you use the link:https://github.com/databricks/sbt-spark-package[sbt-spark-package plugin], in your sbt build file, add:
+
+[shell]
+----
+scala spDependencies += "neo4j-contrib/neo4j-connector-apache-spark_${scala.version}_${spark.version}:4.0.1"
+----
+
+Otherwise:
+
+[text]
+----
+resolvers += "Spark Packages Repo" at "http://dl.bintray.com/spark-packages/maven"
+libraryDependencies += "neo4j-contrib" % "neo4j-connector-apache-spark_${scala.version}_${spark.version}" % "4.0.1"
+----
+
+=== Using Maven
+
+.pom.xml
+[source,xml]
+----
+<dependencies>
+  <!-- list of dependencies -->
+  <dependency>
+    <groupId>neo4j-contrib</groupId>
+    <artifactId>neo4j-connector-apache-spark_${scala.version}_${spark.version}</artifactId>
+    <version>4.0.1 </version>
```

Comment:
Thanks for doing this PR!

Just change 4.0.1 here to something like ${connector.version} and PR is approved / mergeable.

These docs will age in 6 months and we'll be on a different version, but users will eternally lazy copypasta!